packages/client3/pages/artist/pool/[address]/artist-pool-information.tsx

```diff
@@ -88,56 +93,44 @@ function ArtistWithdrawPool({
               {numOfBackers > 1 ? "Backers" : "Backer"}
             </BodyText>
           </div>
+          {!lockedPool ? (
+            <div className="mt-8 w-full border border-dark-90" />
+          ) : (
+            <></>
```

Comment:
You can also return null instead of React fragment(<></>)

Reply:
```suggestion
            null
```

Reply:
Ahh thank you!!

Reply:
Could be added to other such returns as well